app/embeds/GoogleDrawings.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as React from "react";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+  "^https://docs.google.com/drawings/d/(.*)/(edit|preview)(.*)$"
+);
+
+type Props = {|
+  attrs: {|
+    href: string,
+    matches: string[],
+  |},
+|};
+
+export default class GoogleDrawings extends React.Component<Props> {
+  static ENABLED = [URL_REGEX];
+
+  render() {
+    return (
+      <Frame
+        {...this.props}
+        src={this.props.attrs.href.replace("/edit", "/preview")}
+        icon={
+          <img
```

Comment:
Please pull in `develop` and use the `Image` component here now, it added support for loading images from a CDN since this fork was created. see:

https://github.com/outline/outline/blob/develop/app/embeds/GoogleDocs.js#L24

Reply:
done :)